docs/components/modeler/web-modeler/process-application-pipeline.md

```diff
@@ -0,0 +1,92 @@
+---
+id: process-application-pipeline
+title: Process application development pipeline
+description: The Web Modeler provides a lightweight development pipeline you can use to quickly develop and progress low-risk process application releases through the stages of a standard development life cycle.
```

Comment:
This doesn't match my mental image of how the deployment pipeline fits as a part of the development lifecycle. The deployment pipeline does not affect other activities like design, implementation, or maintenance